wordpress-mobile/WordPress-iOS
# Add Jetpack powered badge to Me and App Settings

Fixes #NA

This PR adds the "Jetpack powered" badge to the Me and App Settings screens (see screenshots).
It also fixes the distance of the badge from the top in activity detail, to align it with the designs and the other badges

Me iPad Dark | App Settings iPhone Light
-- | --
<img width=500 src=https://user-images.githubusercontent.com/34376330/179604684-12e053c3-4689-429e-9a4e-213b4c4b5b9f.png> | <img width=400 src=https://user-images.githubusercontent.com/34376330/179604703-e7187c19-f24e-4766-89a6-f1f2846e0ac2.png>



To test:

- Build/run the WordPress target on this branch
- Go to Me and make sure the "Jetpack powered" badge appears at the bottom
- Tap App Settings and make sure the "Jetpack powered" badge appears at the bottom
- Stop execution and build the Jetpack target
- Make sure the banner does not appear in both cases listed above
- Stop the execution, turn the `jetpackPowered` feature flag to false
- Re-build/run the WordPress target and make sure the banner does not appear on the same cases

## Regression Notes
1. Potential unintended areas of impact
Low, adds a row to both Me and App Settings

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Test on several devices

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:
